File: exercpythonbrasil/1.exerc_sequencial/exerc11.py
"""
Faça um Programa que peça 2 números inteiros e um número real. Calcule e mostre:
A. o produto do dobro do primeiro com metade do segundo .
B. a soma do triplo do primeiro com o terceiro.
C. o terceiro elevado ao cubo.
"""
num = int(input("Digite o primeiro numero: "))
num2 = int(input("Digite o segundo numero: "))
num3 = float(input("Digite o terceiro numero: "))

# A.
a = (num * 2) * (num2/2)
print("o produto do dobro do primeiro com metade do segundo:",a)

# B.
b = (num * 3) + num3
print("a soma do triplo do primeiro com o terceiro:",b)

# C.
c = num3 ** 2
print("o terceiro elevado ao cubo:",c)

File: exercicios_C/1.exercSequenciais/exerc11.c
#include <stdio.h>
#include <math.h>

void main(){
    int a, b = 0;
    float c, result_a, result_b, result_c = 0;
    printf("informe o primeiro numero inteiro: ");
    scanf("%d", &a);
    printf("informe o segundo numero inteiro: ");
    scanf("%d", &b);
    printf("informe o numero decimal: ");
    scanf("%f", &c);
    result_a = (a * 2) * ((float)b / 2);
    result_b = (a * 3) + c;
    printf("o produto do dobro do primeiro com metade do segundo: %0.1f.\n", result_a);
    printf("a soma do triplo do primeiro com o terceiro: %0.1f.\n", (float)result_b);
    printf("o terceiro elevado ao cubo: %0.2f.", pow(c,3));
}
